server/entities/chat/Chat.ts

```diff
@@ -18,6 +18,16 @@ const chatSchema = new Schema({
       required: true,
     },
   ],
+  name: {
```

Comment:
tu będzie konflikt z moimi zmianami, 

jest field 'chatName' który jest nieobowiązkowy (tj, nullish)
więc nie wiem czy chcemy narzucać required + unique, nie są to żadne klucze więc można pozwolić na duplikaty o tej samej nazwie (ale np innym zdjęciu!) 

bo np dla direct chatów ciężko wydedukować jaka nazwa powinna być (jeśli jest nullish to partner rozmowy, jesli non-nullish to nazwa wlasna)

pasuje też dopisać możliwość edycji czatu (nazwa, zdjęcie) chyba że to już jest

Reply:
ja myślałem, że nie robimy ostatecznie edycji, ale nawet jeśli to można w osobnym prze